extensions/email/src/main/java/org/mule/extension/email/api/EmailConnectionSettings.java

```diff
@@ -23,21 +25,27 @@
      * The host name of the mail server.
      */
     @Parameter
+    @Placement(tab = "General", group = "Connection", order = 1)
```

Comment:
looksl ike things like General or connection are going to be pretty common across extensions. They're even very repeated here. Define those values as constants in the annotation itself